CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## CHANGELOG for Wale's pw generator
+This file is used to list changes made in each version of the pw generator 
+
+## 0.1 
```

Comment:
I have established, started the version train to start at 0.1, but you could alternatively change this to like 1.0.1, which would include major, minor, and patch versions and is more verbose. Or, 1.1, or whatever.